TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/README.md

```diff
@@ -32,8 +32,8 @@ metrics_source:
 
 Source and target clusters have the following options:
 - `endpoint`: required, the endpoint to reach the cluster.
-- `authorization`: optional, if it is provided, type is required.
-	- `type`: required, the only currently implemented option is "basic", but "sigv4" should be available soon
+- `authorization`: required, the auth method to use, if no auth the no_auth type must be specified.
+	- `type`: required, the only currently implemented options are "no_auth" and "basic", but "sigv4" should be available soon
```

Comment:
mind updating this to `"basic_auth"`? (in the example yaml above as well)

Reply:
Updated these spots now